fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -548,6 +548,9 @@ def state_dict(self, *args: Any, **kwargs: Any) -> "OrderedDict[str, torch.Tenso
             self._cast_buffers()
         return state_dict
 
+    def state_dict(self, *args: Any, **kwargs: Any) -> "OrderedDict[str, torch.Tensor]":  # type: ignore
+        return self._state_dict(*args, **kwargs)
```

Comment:
could you please add a comment on why is this wrapper needed? without it, it is easily for future refactor to undo this change.

Reply:
Sure I will add a comment - good point